pykafka/balancedconsumer.py

```diff
@@ -509,6 +525,7 @@ def _consumers_changed(self, consumers):
         if self._setting_watches:
             return
         log.debug("Rebalance triggered by consumer change")
+        self._add_self()
```

Comment:
Could this get a short comment with it? I don't fully grasp what situation it's needed in.